src/Microsoft.DotNet.Build.CloudTestTasks/AzureHelper.cs

```diff
@@ -0,0 +1,373 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="AzureHelper.cs" company="">
+//   
+// </copyright>
+// <summary>
+//   The azure helper.
+// </summary>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace Microsoft.DotNet.Build.CloudTestTasks
+{
+    #region
+
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Net;
+    using System.Security.Cryptography;
+    using System.Text;
+    using System.Text.RegularExpressions;
+
+    #endregion
+
+    /// <summary>
+    /// The azure helper.
+    /// </summary>
+    public static class AzureHelper
+    {
+        /// <summary>
+        /// The storage api version.
+        /// </summary>
+        public static readonly string StorageApiVersion = "2015-04-05";
+
+        /// <summary>
+        /// The sas access type.
+        /// </summary>
+        public enum SasAccessType
+        {
+            /// <summary>
+            /// The read.
+            /// </summary>
+            read, 
+
+            /// <summary>
+            /// The write.
+            /// </summary>
+            write
```

Comment:
It's a good idea to always end in trailing commas (when the language allows), so that appending things in the future doesn't show an edit on this line.  (`git blame` and other "when did this show up?" type tools will show the original addition, instead of "oh, adding a comma")